modules/restful_token_auth/plugins/restful/restful_token_auth/token_auth/refresh_token/1.0/RestfulRefreshTokenAuthentication.class.php

```diff
@@ -47,6 +47,22 @@ public function refreshToken($token) {
     // Remove the refresh token once used.
     $refresh_token = entity_load_single('restful_token_auth', key($results['restful_token_auth']));
     $uid = $refresh_token->uid;
+
+    // Get the access token linked to this refresh token then do some cleanup.
+    $accessTokenQuery = new EntityFieldQuery();
+    $accessTokenReference = $accessTokenQuery
```

Comment:
Please change `$accessTokenReference` to `$access_token_reference` as per Drupal Coding Standards.